docs/guides/faq.md

```diff
@@ -0,0 +1,91 @@
+## FAQ
+
+**Q: What is video.js?**
+> video.js is an extendable wrapper around the native video element. It does the following:
+>  * Offers a plugin API so that different types of video can be handed to the native video element. IE:  HLS, Flash, or HTML5 video.
+>  * Polyfills and unifies the native video api across browsers
+>  * Offers an extendable and themable UI
+>  * Takes care of accessibility for the user (in-progress)
+
+**Q: How do I install video.js?**
+> Currently video.js can be installed using bower, npm, or by serving a release file
+> from a github tag. For information on doing any of those see the [install guide](http://videojs.com/getting-started/).
+
+**Q: What do video.js version numbers mean?**
+> video.js follows [semver](http://semver.org/)
+
+**Q: How can I troubleshoot playback issues?**
+> See the [troubleshooting guide](/docs/guides/troubleshooting.md)
```

Comment:
I think the byte range/php hosting question need to be in this FAQ, unless the troubleshooting guide is going to be an FAQ of more technical topics.